internal/cmd/generate/commands/gentests/generator.go

```diff
@@ -144,7 +144,6 @@ func (g *Generator) gorun(code string) (string, error) {
 		os.RemoveAll(dir)
 	}()
 
-	// fmt.Println(code)
```

Comment:
Thanks for looking into these parts, but frankly, the generator code is so hairy, that it first needs some proper refactoring... Can you skip the `internal/cmd` parts in the patch?

Reply:
Done!